file-formats/intm/type/zif_aff_intm_v1.intf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "General types reusable in AFF",
```

Comment:
This is the description of the AFF itself, so what does this AFF do? What does INTM stand for? Intelligent Model?

Reply:
This description is not part of our interface.